clients/java/src/main/java/io/camunda/client/impl/http/ApiEntityConsumer.java

```diff
@@ -83,7 +83,7 @@ protected ApiEntity<T> generateContent() throws IOException {
 
   @Override
   protected int capacityIncrement() {
-    return maxCapacity - entityConsumer.getBufferedBytes();
+    return chunkSize;
```

Comment:
:question: So we still silently ignore when responses would be too large right? :(

Reply:
No, with this change the client is always able to receive more data, in chunks of `chunkSize`.

We should probably put a hard limit (like 4x chunkSize) and throw an exception if that is breached.

Reply:
Ah, yeah, got it :+1: